tests/test_schema.py

```diff
@@ -22,7 +22,7 @@
 
 from pydax import export_schemata, init, load_schemata
 from pydax.schema import Schema, SchemaManager
-from pydax._schema import get_schemata
+from pydax._high_level import get_schemata
```

Comment:
Should `get_schemata` be imported from `pydax` instead of from `pydax._high_level`?

Reply:
We don't expose `get_schemata`, see my other comment